mix.exs

```diff
@@ -32,7 +32,8 @@ defmodule Scidata.MixProject do
     [
       {:ex_doc, ">= 0.24.0", only: :dev, runtime: false},
       {:nimble_csv, "~> 1.1"},
-      {:jason, "~> 1.0"}
+      {:jason, "~> 1.0"},
+      {:stb_image, "~> 0.1.0", github: "cocoa-xu/stb_image", optional: true}
```

Comment:
Just a heads up: we will need to convert it to a package before a package release (as packages cannot have git deps).

Reply:
Thanks for the reminder @josevalim!

@cocoa-xu would you be willing to publish `stb_img` on hex.pm?

Reply:
Of course! I just published it is on hex.pm.

As a side note, it seemed that publishing to hex.pm will remove the git submodule info (not sure which files are necessary to keep the submodule info, and I'm not sure if I should go with the submodule way when publishing to hex.pm). Therefore, I downloaded the `stb_image.h` and its LICENSE file and gave up using git submodule.

```suggestion
      {:stb_image, "~> 0.1.2", optional: true}
```

I tested this version on my machine, and it worked as expected.

Reply:
Thank you @cocoa-xu for your critical help here. :)

Reply:
Not at all! I'm glad I could be of help! 😄